Sming/Libraries/MPU6050/README.md

```diff
@@ -0,0 +1,8 @@
+# MPU6050 Six-Axis (Gyro + Accelerometer)
+
+Based on code from [jrowberg/i2cdevlib](https://github.com/jrowberg/i2cdevlib/tree/master/ESP32_ESP-IDF/components/MPU6050). Most of the code is the same, except:
```

Comment:
Maybe date and which commit this is based on.

Can also add doxygen documentation, but requires this file to be renamed to `README.rst` and heading/url adjusted accordingly.

```
API Documentation
-----------------

.. doxygenclass:: MPU6050
   :members:

```